utils/average_checkpoints.py

```diff
@@ -74,7 +74,10 @@ def main():
         # average
         for k in avg.keys():
             if avg[k] is not None:
-                avg[k] /= args.num
+                if avg[k].is_floating_point():
+                    avg[k] /= args.num
+                else:
+                    avg[k] //= args.num
```

Comment:
Probably casting is better?
I'm concerning the issue of "//" in https://en.wikibooks.org/wiki/Python_Programming/Operators

Reply:
The int tensor in conformer seems to be something to do with kernel or positional information that's being saved. So I think might be the same value that's being averaged across checkpoints which case -- // shouldn't cause an issue.